test/src/test/java/hudson/cli/ClientAuthenticationCacheTest.java

```diff
@@ -114,6 +114,29 @@ public void getPropertyKey() throws Exception {
         String key = cache.getPropertyKey();
         assertTrue(key, Secret.decrypt(key) != null);
     }
+    
+    @Test
+    @Issue("JENKINS-47426")
```

Comment:
IIUC this passed before, as a control, so is not really part of this issue. Maybe better to just merge the two test cases.

Reply:
merged as proposed